src/script/common/c_content.cpp

```diff
@@ -1392,3 +1402,41 @@ void read_json_value(lua_State *L, Json::Value &root, int index, u8 recursion)
 	}
 	lua_pop(L, 1); // Pop value
 }
+
+void push_pointed_thing(lua_State *L, const PointedThing &pointed)
+{
+	lua_newtable(L);
+	if(pointed.type == POINTEDTHING_NODE)
+	{
+		lua_pushstring(L, "node");
```

Comment:
please fix the code style (spaces before if and braces at the end of the condition